src/main/java/games/strategy/triplea/ui/screen/Tile.java

```diff
@@ -32,55 +32,55 @@
   public static final LockUtil S_TILE_LOCKUTIL = LockUtil.INSTANCE;
   private static final boolean DRAW_DEBUG = false;
   private static final Logger s_logger = Logger.getLogger(Tile.class.getName());
+
   // allow the gc to implement memory management
-  private SoftReference<Image> m_imageRef;
-  private boolean m_isDirty = true;
-  private final Rectangle m_bounds;
-  private final int m_x;
-  private final int m_y;
-  private final double m_scale;
-  private final Lock m_lock = new ReentrantLock();
-  private final List<IDrawable> m_contents = new ArrayList<>();
+  private SoftReference<Image> imageRef;
+  private boolean isDirty = true;
+  private final Rectangle bounds;
+  private final int x;
+  private final int y;
```

Comment:
The `x` and `y` fields still trigger a violation because fields names must be at least two characters long.  I think these field names are perfectly valid in this case, as well as in a few other geometry-focused classes.  (I did consider renaming them to `left` and `top`, respectively, but that would have required renaming methods, as well, which would have triggered a lot of unrelated changes outside the scope of this PR.)

I plan on submitting an issue soon to discuss how we should proceed suppressing individual Checkstyle warnings where it makes sense without having to modify the global Checkstyle configuration.